src/CodeFormatter/Program.cs

```diff
@@ -111,17 +152,32 @@ private static int Main(string[] args)
 
             try
             {
-                RunAsync(
-                    projectOrSolutionPath,
-                    ruleTypeBuilder.ToImmutableArray(),
-                    fileNamesBuilder.ToImmutableArray(),
-                    configBuilder.ToImmutableArray(),
-                    copyrightHeader,
-                    allowTables,
-                    convertUnicode,
-                    verbose,
-                    ct).Wait(ct);
-                Console.WriteLine("Completed formatting.");
+                var rules = ruleTypeBuilder.ToImmutableArray();
+                var fileNames = fileNamesBuilder.ToImmutableArray();
+                var config = configBuilder.ToImmutableArray();
+
+                foreach (string projectOrSolution in projectOrSolutionPath)
+                {
+                    RunAsync(
+                        projectOrSolution,
+                        rules,
+                        fileNames,
+                        config,
+                        copyrightHeader,
+                        allowTables,
+                        convertUnicode,
+                        verbose,
+                        ct).Wait(ct);
+                }
+
+                if (projectOrSolutionPath.Length > 1)
+                {
+                    Console.WriteLine("Completed formatting {0} projects/solutions.", projectOrSolutionPath.Length);
+                }
+                else
+                {
+                    Console.WriteLine("Completed formatting.");
+                }
```

Comment:
We should move the status line up to be inside the loop.  That way the user gets feedback for every project / solution that is formatted.